### DISCUSSION TITLE
Landing Zone Accelerator

### DISCUSSION THREAD

**Discussion by amirasyraf** [superwerker/superwerker#347] [Q&A] [answered]
What's the difference or advantages/disadvantages of superwerker over [Landing Zone Accelerator](https://aws.amazon.com/solutions/implementations/landing-zone-accelerator-on-aws/)? Both are used to deploy and setup a multi-account environment following various best practices.

**Comment by mavogel** [comment, marked as answer]
Great @amirasyraf for pointing this out 🙏🏾 

I took a look at the docs, videos and this is my comparison:

| Topic: | superwerker  | Landing Zone Accelerator (LZA) |
| -------------| ------------- | ------------- |
| Automatic Controltower (CT)  | :white_check_mark:  | :x: (needs to be installed [manually](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/prerequisites.html))   | 
| CT [drift detection](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/account-creation-and-drift-detection.html)  | :x:  | :white_check_mark: | 
| Security Hub  | :white_check_mark:  | :white_check_mark:  |
| Guardduty  | :white_check_mark:  | :white_check_mark:  |
| Secure RootMail Boxes  | :white_check_mark:  | :x:  |
| Budgets  | :white_check_mark:  | :white_check_mark: (more advanced)  |
| Code Pipeline  | :x:  | :white_check_mark:  |
| Backups  | :white_check_mark:  | :white_check_mark: (more advanced)   |
| Advanced Network config  | :x:  | :white_check_mark:  |
| Sets up AWS acc and creates OUs  | :x:  | :white_check_mark:  |
| cdk  | (:white_check_mark:) *in progress* #328  | :white_check_mark:  |
| Support from AWS in console  | :x:  | :white_check_mark:  |
| open source  | :white_check_mark:  | :white_check_mark:  |
| Use cases  | all industries  | mainly regulated industries  |
| brownfield usable   | *someday* #58   | *toggle it in* see question [here](https://youtu.be/HaXzfNfJR7c?t=2886)  |
| Monthly Cost  | 5$  | [527$](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/cost.html) for recommended config (however dunno with minimal one)  |

Generally this is what we were/are aiming for with superwerker, e.g. *opinionated environments* in #319 

You are right and superwerker is best practice in our opinion and the LZA from AWS' opinion. However, when we started building superwerker back in June 2020, there was already [ControlTower](https://docs.aws.amazon.com/controltower/latest/userguide/January-December-2019.html#Version-2-1), however no such thing, like automatically adding SHub and GD member accounts etc. So we built it back in the time 😃 

What we will definitely to is take a look the packages, such as [accelerator](https://github.com/awslabs/landing-zone-accelerator-on-aws/tree/main/source/packages/%40aws-accelerator/accelerator) (and wait until they might be [released](https://www.npmjs.com/search?q=aws-accelerator%2Faccelerator)...
